src/components/Table/StockTable.jsx

```diff
@@ -25,7 +25,7 @@ const StockTable = ({ stocksData }) => {
                   <TableRow
                     key={stock.symbol}
                     component={Link}
-                    to={'/' + stock.symbol}
+                    to={'/' + stock!=null?stock.symbol:""}
```

Comment:
Can just be written as to={stock?.symbol} (optional chaining for null safety)

Reply:
Thanks , will change it asap

Reply:
I have added the null check at the top now .